### PR TITLE
Do 113 handle exception for invalid url

### PR DESCRIPTION
**Pull request by sandeep-pepcus**

### CODE REVIEW

**Comment by sandeep-pepcus** [comment]
Added new environment variable. Please add this in respective environment.

File: src/main/java/com/pepcus/directoryonline/api/exception/ApiExceptionHandler.java [hunk 1]
```diff
@@ -116,11 +118,24 @@ public ApiError handleUploadFileSizeExceedException(MaxUploadSizeExceededExcepti
...
+  
+  /**
+   * Method to handle url not found exception
+   * @param ex
+   * @return
+   */
+  @ExceptionHandler(NoHandlerFoundException.class)
+  @ResponseStatus(NOT_FOUND)
+  @ResponseBody
+  public ApiError handleNoHandlerFoundException(NoHandlerFoundException ex) {
```
Comment (pepcus-ankit) [inline src/main/java/com/pepcus/directoryonline/api/exception/ApiExceptionHandler.java:135]: I guess if we are handling this in handler than we don't need to do changes in application context to enable it.

Comment (sandeep-pepcus) [reply src/main/java/com/pepcus/directoryonline/api/exception/ApiExceptionHandler.java:135]: default value of 'spring.mvc.throw-exception-if-no-handler-found' is false. So when url is not match with any url of application it return directly from application level and not came at handler layer. It return /api/error. default

Comment (sandeep-pepcus) [reply src/main/java/com/pepcus/directoryonline/api/exception/ApiExceptionHandler.java:135]: If value of spring.mvc.throw-exception is true then it is not handled by framework and available to handle manually at handler level.

**Pull request merged**